BlockServer/core/database_client.py

```diff
@@ -0,0 +1,14 @@
+from server_common.utilities import dehex_and_decompress, print_and_log
+from server_common.channel_access import ChannelAccess
+from server_common.pv_names import DatabasePVNames
+import json
+
+
+def get_iocs():
+    # Get the list of available IOCs from DatabaseServer
+    try:
+        rawjson = dehex_and_decompress(ChannelAccess.caget(DatabasePVNames.IOCS))
```

Comment:
This pv is missing the prefix so it can not be found I see:

`[Thu Dec 20 17:50:43 2018] [1545328243.91] MAJOR: Could not retrieve IOC list: 'NoneType' object has no attribute 'decode'`

Sympton is that IOC are never stopped on load config.